stacklight_tests/clients/influxdb_grafana_api.py

```diff
@@ -161,113 +198,325 @@ def get_all_measurements(self):
         return measurements
 
 
+class InfluxDBQueryBuilder(object):
+    def __init__(self, source):
+        self.target = source
+        self.tags = source["tags"]
+        self.select = source["select"]
+        self.policy = source.get("policy", "default")
+        self.measurement = source["measurement"]
+        self.group_by = source["groupBy"]
+
+    def _render_tags(self):
+        res = []
+        for tag in self.tags:
+            value = tag["value"]
```

Comment:
It's required field, why should I take nothing instead of something? I think it should be exception (earlier - better), if structure has invalid format.